src/index.js

```diff
@@ -2,8 +2,10 @@ export function download(url, fullName) {
   const anchor = document.createElement('a')
   anchor.href = url
   anchor.setAttribute('download', fullName)
-
-  anchor.click();
+  document.body.appendChild(anchor)
+  anchor.click()
+  document.body.removeChild(anchor)
+  window.URL.revokeObjectURL(url)
```

Comment:
I'm not quite familiar with this API. Could you please elaborate on why should we call it after the DOM node being removed?

Reply:
You mean this line?
```
window.URL.revokeObjectURL(url)
```

I'm sorry, I'll remove the last line, since it's not responsibility of the container function to create the `url`. Should be called in case that url is created through [createObjectUR](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL)

Reply:
`appendChild` before `click` seems to be necessary to make it work on FF, otherwise doesn't force the _download_

Reply:
Also can we make it `display: none` even it's removed shortly afterwards?

Reply:
Sure! As I did [here](https://github.com/elrumordelaluz/micro-svg-spreact/commit/4617e8da0f5a65f56a8a4341f78c8e8e871c2e0e#diff-f606ad2265472a3cdd48eb8a47161e98). I'm sorry to missing that